### PR TITLE
III-6264 acceptance test default filter from UiTID

### PR DESCRIPTION
**Pull request by JonasVHG**
### Added

- `default-queries-uitid.feature`: An acceptance test for default queries from `UiTID`

### Related PR

- https://github.com/cultuurnet/appconfig/pull/755

---
Ticket: https://jira.publiq.be/browse/III-6264

### CODE REVIEW

File: features/search/default-queries-uitid.feature [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  Scenario: Search for a place blocked by the default query
+    Given I create a place from "places/hemmekes.json" and save the "id" as "placeId"
+    And I am using the Search API v3 base URL
+    And I am not authorized
+    And I am using an UiTID v1 API key of consumer "sapi3KeyWithUitIdFilterForDiest"
+    And I am not using a x-client-id header
+    And I wait 2 seconds
+    When I send a GET request to "/places" with parameters:
+      | limit                 | 1 |
+      | embed                 | true |
```
Comment (LucWollants) [inline features/search/default-queries-uitid.feature:19]: Nit: no need to pass embed

File: features/search/default-queries-uitid.feature [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    And I am authorized as JWT provider v1 user "centraal_beheerder"
+    And I send and accept "application/json"
+
+  Scenario: Search for a place blocked by the default query
+    Given I create a place from "places/hemmekes.json" and save the "id" as "placeId"
+    And I am using the Search API v3 base URL
+    And I am not authorized
+    And I am using an UiTID v1 API key of consumer "sapi3KeyWithUitIdFilterForDiest"
+    And I am not using a x-client-id header
+    And I wait 2 seconds
```
Comment (LucWollants) [inline features/search/default-queries-uitid.feature:16]: You could avoid this `wait`, if you use `I wait for the place with URL "%{placeUrl}" to be indexed` after the `Given`

File: features/search/default-queries-uitid.feature [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    Given I am using the UDB3 base URL
+    And I am using an UiTID v1 API key of consumer "uitdatabank"
+    And I am authorized as JWT provider v1 user "centraal_beheerder"
+    And I send and accept "application/json"
+
+  Scenario: Search for a place blocked by the default query
+    Given I create a place from "places/hemmekes.json" and save the "id" as "placeId"
+    And I am using the Search API v3 base URL
+    And I am not authorized
+    And I am using an UiTID v1 API key of consumer "sapi3KeyWithUitIdFilterForDiest"
```
Comment (LucWollants) [inline features/search/default-queries-uitid.feature:14]: The default query that gets applied here is a bit hidden, maybe we need to put it in a comment somewhere? Now it's a bit hard to figure out why the place gets filtered out

Comment (JonasVHG) [reply features/search/default-queries-uitid.feature:16]: I have put it in this file, alternative might be to put it in comments in `config.features.php`

**Review by LucWollants — APPROVED** [review]
Thanks for the extra tweaks!

**Pull request merged**